lib/ansible/modules/cloud/amazon/ec2_eni.py

```diff
@@ -100,6 +100,12 @@
         This option is mutually exclusive of secondary_private_ip_address_count
     required: false
     version_added: 2.2
+  purge_secondary_private_ip_addresses:
+    description:
+      - To be used with I(secondary_private_ip_addresses) to determine whether or not to remove any secondary IP addresses other than those specified.
```

Comment:
Is it worth mentioning that secondary_private_ip_addresses should be set to the empty list if all secondary addresses should be purged.

Reply:
Addressed.